modules/glib/2.82.2.bcr.2/MODULE.bazel

```diff
@@ -0,0 +1,31 @@
+module(
+    name = "glib",
+    version = "2.82.2.bcr.2",
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+bazel_dep(name = "libffi", version = "3.4.7")
+bazel_dep(name = "pcre2", version = "10.46-DEV")
```

Comment:
I'm guessing the use of a `-DEV` version here is due to https://github.com/PCRE2Project/pcre2/pull/729 -- can you add a comment to that effect?